api/notd/token_metadata_processor.py

```diff
@@ -32,6 +32,15 @@ class TokenDoesNotExistException(NotFoundException):
 class TokenHasNoMetadataException(NotFoundException):
     pass
 
+class DefaultTokenMetadata():
+
+    def set_default(self, registryAddress: str, tokenId: str):
+        return RetrievedTokenMetadata(
```

Comment:
this is very wierd - just inline this where you are using it. No need for a class or a function